dev-libs/girara/girara-0.4.4.ebuild

```
@@ -0,0 +1,62 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson virtualx
+
+DESCRIPTION="UI library that focuses on simplicity and minimalism"
+HOMEPAGE="https://pwmt.org/projects/girara/"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.pwmt.org/pwmt/${PN}.git"
+	EGIT_BRANCH="develop"
+else
+	SRC_URI="https://github.com/pwmt/girara/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="amd64 arm ~arm64 ~riscv x86"
+fi
+
+LICENSE="ZLIB"
+SLOT="0"
```

This is going to be annoying.
```
 SONAME:-libgirara-gtk3.so.3(64)
 SONAME:+libgirara-gtk3.so.4(64)
------> FILES(+18,-18) SONAME(+1,-1)
```
so from girara-0.4.2 to 0.4.4 we got a SONAME bump. Pretty much all girara's rev deps need to be updated for this. I can do it if you update the the rest of my reviewed parts.


```suggestion
SLOT="0/4"
```